src/patharray.js

```diff
@@ -100,6 +100,63 @@ SVG.extend(SVG.PathArray, {
 
     return this
   }
+  // Test if the passed path array use the same commands as this path array
+, haveSameCommands: function(pathArray) {
```

Comment:
Not sure why this is needed. Are we looking for a base class, if so then we could use `instanceof` or do we want to duck type `pathArray`?